src/main/java/org/cascadebot/cascadebot/commands/core/InviteCommand.java

```diff
@@ -29,7 +29,7 @@ public String command() {
 
     @Override
     public String description() {
-        return "Gets the bot invite link";
+        return "Returns the bot invite link";
```

Comment:
We can be more specific with this, invite link to what?

Reply:
it's pretty obvious, returns the bot invite link so that you can invite the bot to the server

Reply:
Obvious to you, some people might interpret it as a server invite link 🤷‍♂️